js-packages/@prestojs/ui-antd/src/AntdUiProvider.tsx

```diff
@@ -137,52 +135,127 @@ export function useAntdUiConfig(): AntdUiConfig {
 }
 
 /**
- * [antd](https://ant.design) specific version of [UiProvider](doc:UiProvider). Provides default antd components to use
- * to [UiProvider](doc:UiProvider) and supports some extra options specific to antd.
+ * @expand-properties
+ */
+export type AntdUiProviderProps = {
+    /**
+     * The DatePicker component to use for components like [DateWidget](doc:DateWidget). If you don't use any of
+     * these components you don't have to provide this option.
+     *
+     * You can pass the antd [DatePicker](https://ant.design/components/date-picker/) directly
+     * or [create your own version](https://ant.design/docs/react/replace-moment#DatePicker)
+     */
+    datePickerComponent?: DatePickerComponent<any>;
+    /**
+     * The TimePicker component to use for components like [TimeWidget](doc:TimeWidget). If you don't use any of
+     * these components you don't have to provide this option.
+     *
+     * You can pass the antd [TimePicker](https://ant.design/components/time-picker/) directly
+     * or [create your own version](https://ant.design/docs/react/replace-moment#TimePicker)
+     */
+    timePickerComponent?: TimePickerComponent<any>;
+} & UiProviderProps;
+
+/**
+ * Version of [UiProvider](doc:UiProvider) that supports some extra options specific to antd.
+ *
+ * If you use date or time widgets in the system you should pass through the `datePickerComponent` and `timePickerComponent`
+ * props. Without this when [DateWidget](doc:DateWidget), [TimeWidget](doc:TimeWidget) or other date/time based components
+ * are used they will throw an error. The reason for this is to configure how you want dates to be handled. The default
+ * provided by antd is to use momentjs but you may wish to switch this to date-fns or dayjs. To use the default (moment)
+ * based components do:
```

Comment:
(same comment as b4: we might want to use stronger word to recommend against moment)